rosie/core/__init__.py

```diff
@@ -28,7 +28,10 @@ def __init__(self, settings, adapter):
         self.settings = settings
         self.dataset = adapter.dataset
         self.data_path = adapter.path
-        self.suspicions = self.dataset[self.settings.UNIQUE_IDS].copy()
+        if self.settings.UNIQUE_IDS:
+            self.suspicions = self.dataset[self.settings.UNIQUE_IDS].copy()
+        else:
+            self.suspicions = self.dataset.copy()
```

Comment:
Do we need consistency in this unique identifiers? I mean, considering Rosie runs today and tomorrow: is it really required that the document X today have exactly the same id as it would have tomorrow? If not we can bring pandas index (created by default) to the dataset.